packages/angular/src/table/head/table-head-cell.scss

```diff
@@ -0,0 +1,42 @@
+// TODO add styling hooks to carbon-components-angular so we can remove aiTableHeadCell
+.table-head-cell-text,
+.table-head-cell-icons {
+  top: 0;
+}
+
+.bx--data-table--compact {
```

Comment:
you may want to use prefix variables as Carbon is doing away with the `bx` in major release.

Reply:
Ah, yes 👍